src/navbar.js

```diff
@@ -0,0 +1,38 @@
+import { Link } from 'react-router-dom';
+
+const Navbar = () => (
+  <nav className="nav-bar">
+    <Link
+      to="/"
+      className="logo"
+      style={{ color: 'rgb(1, 254, 1)', fontStyle: 'italic', fontSize: '1.6rem' }}
+    >
+      Math
+      <span> Magicians </span>
+    </Link>
+    <ul>
+      <li className="active">
+        <Link to="/">Home</Link>
+      </li>
+      <li>
+        <Link to="/calculator">Calculator</Link>
+      </li>
+      <li>
+        <Link to="/quote">Quote</Link>
+      </li>
+    </ul>
+  </nav>
+);
+
+// function CustomLink({ href, children, ...props }) {
+//   const path = window.location.pathname;
+//   return (
+//     <li className={path === href ? 'active' : ''}>
+//       <a href={href} {...props}>
+//         {children}
+//       </a>
+//     </li>
+//   );
+// }
```

Comment:
- [ ] Kindly remove this commented code to make your code clean from any unused code.